metrics/types/types.proto

```diff
@@ -39,3 +39,11 @@ message ViewTimeouts {
   // Number of view timeouts.
   uint64 Timeouts = 3;
 }
+
+message CPUMemoryStats {
```

Comment:
Suggest renaming this to `CPUMemoryMeasurement` to align with the `Throughput` and `Latency` variants.